nexusadspy/client.py

```diff
@@ -125,7 +125,6 @@ def _do_throttled_request(self, url, method, params=None, data=None, headers=Non
                               sec_sleep=2., max_failures=100,
                               get_field=None):
 
-        data = json.dumps(data)
```

Comment:
But requests library has in-built capability to accept string, dict or bytes as data parameter https://github.com/kennethreitz/requests/blob/master/requests/api.py#L23 . Any reason we plan to handle it at our side again?